src/Seq.Forwarder/Cli/Commands/InstallCommand.cs

```diff
@@ -188,6 +190,7 @@ void Install(TextWriter cout)
                 "/LogFile=\"\"",
                 "/ShowCallStack",
                 "/storage=\"" + _storagePath.StorageRootPath + "\"",
+                "/listen=\"" + _listenUri.ListenUri + "\"",
```

Comment:
Instead of passing this through as a service parameter, it'd be preferable to write it into the `SeqForwarderConfig`  opened a few lines earlier. This gets rid of quite a bit of extra machinery - `RunCommand` can just load it from the config rather than parse the installed service command-line.

(Seq originally passed `--listen` through to the service as well, but reinstalling turned out to be brittle, so all configuration is now in the config file, apart from the location of the config file itself.)